### PR TITLE
EUI-7361

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-7361

### Change description ###
Changes to support tab

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by johnbenjamin-hmcts — APPROVED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/read-case-flag-field.component.spec.ts [hunk 1]
```diff
@@ -387,6 +387,7 @@ describe('ReadCaseFlagFieldComponent', () => {
     fixture = TestBed.createComponent(ReadCaseFlagFieldComponent);
     component = fixture.componentInstance;
     component.caseField = flagLauncher1CaseField;
+    component.formGroup = new FormGroup({});
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/read-case-flag-field.component.spec.ts:390]: @MateiHMCTS - is this strictly required? It seems the unit tests that need one initialize `component.formGroup` with the `const formGroup` that is declared earlier

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@MateiHMCTS - just a minor thing that needs checking; the rest is OK

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/read-case-flag-field.component.spec.ts (hunk 1 above)
Comment (MateiHMCTS) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/read-case-flag-field.component.spec.ts:390]: Required for the change in [read-case-flag-field.component.ts](https://github.com/hmcts/ccd-case-ui-toolkit/pull/1361/files#diff-d946ca96bcd1ed92266e51ef8ae71957f7c31365e40954cd37aa311909568047) -- lines 41-44

Comment (Daniel-Lam) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/read-case-flag-field.component.spec.ts:390]: Thanks for checking, @MateiHMCTS - happy to mark as resolved

**Review by Daniel-Lam — APPROVED** [review]

**Pull request merged**